projects/go-lib/src/lib/components/go-side-nav/go-nav-group/go-nav-group.component.ts

```diff
@@ -26,6 +26,7 @@ import { NavItem } from '../nav-item.model';
 export class GoNavGroupComponent implements OnInit {
   @Input() navItem: NavGroup | NavItem;
   @Input() class: string;
+  @Input() index: number;
```

Comment:
Is this being used anywhere?

Reply:
It was not, that was from a different solution I explored and forgot to remove. Good catch.